service/local-dev/run_server.sh

```diff
@@ -74,7 +74,7 @@ elif [[ ${useSdUnderlays} ]]; then
   # export TANAGRA_ACCESS_CONTROL_MODEL=VUMC_ADMIN
 elif [[ ${useEmergeUnderlays} ]]; then
   echo "Using eMerge underlay."
-  export TANAGRA_UNDERLAY_FILES=emerge/emerge_20240603_local
+  export TANAGRA_UNDERLAY_FILES=emerge/emerge_20240603_local,sd/sd020230831_local
```

Comment:
Did you mean to add the `sd` underlay to the eMerge block?

Reply:
yes wanted bpth underlays :-)